msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1454,6 +1452,17 @@ When you use these install and uninstall events, there are some instances where
 
 It's important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you're using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you don't choose to handle.
 
+## Handling errors in conversation events
+
+When the bot encounters an error while handling different events or activities, it shouldn't post random messages directly to the conversation, following is an example:
+
+:::image type="content" source="../../../assets/images/error-msg.PNG" alt-text="The screenshot shows an error message to the conversation with no meaningful context.":::
+
+The bot must send meaningful messages in conversations with specific error message for better user experience. For more information, see [handle unexpected errors](/azure/bot-service/bot-builder-howto-handle-user-interrupt).
+
+> [!NOTE]
+> In production environment, you should consider sign in to Azure Application Insights. For more information, see [https://aka.ms/bottelemetry](https://aka.ms/bottelemetry) to add telemetry capture to your bot.
```

Comment:
"consider sign in to Azure Application Insights" -- improve wording. what do you mean by sign in to application insights?

Reply:
In production environment, you should configure Azure Application Insights for events/errors logging. For more information, see [https://aka.ms/bottelemetry](https://aka.ms/bottelemetry) to add telemetry capture to your bot.

Reply:
Added as a Note